### ISSUE TITLE
Toolbar DOM tweaks

### ISSUE THREAD

**Issue by schanzer** [ds26gte/code.pyret.org#80]
* The row of buttons should be inside a `<nav>` element, with the buttons being a `<UL>` element
* Second row of buttons should also be a `<UL>`

**Comment by ds26gte** [comment]
Fixed in e6ad4cf.
